shopfloor/services/zone_picking.py

```diff
@@ -785,7 +786,15 @@ def _set_destination_location(self, move_line, quantity, confirmation, location)
         # destination location set to the scanned one
         self._write_destination_on_lines(move_line, location)
         stock = self._actions_for("stock")
-        stock.mark_move_line_as_picked(move_line, quantity)
+        try:
+            stock.mark_move_line_as_picked(move_line, quantity, check_user=True)
+        except UserError:
```

Comment:
I would use a specific custom exception instead

Reply:
Done

Reply:
Yeah, but you're still catching the UserError

Reply:
You are right I missed one place.